journal.py

```diff
@@ -34,8 +34,8 @@
 
 class Entry(Base):
     __tablename__ = 'entries'
-    
```

Comment:
Removing whitespace and linting cleanup is a good example of a single commit to make that only does that.